Updated vsso [instances, annotation]

danielwilms opened the pull request:
Output of new tooling in https://github.com/COVESA/vss-tools/pull/160

- DVP now instances
- use of `skos:definition` instead of `rdfs:comment`
- `rdfs:comment` where vss uses comments (new in 3.0)
- dot notated path in separate annotation property

Signed-off-by: Daniel Wilms <Daniel.DW.Wilms@bmw.de>

danielwilms committed:
Updated vsso [instances, annotation]

Output of new tooling in https://github.com/COVESA/vss-tools/pull/160

- DVP now instances
- use of `skos:definition` instead of `rdfs:comment`
- `rdfs:comment` where vss uses comments (new in 3.0)
- dot notated path in separate annotation property

Signed-off-by: Daniel Wilms <Daniel.DW.Wilms@bmw.de>

felix-loesch commented:
Great to see that DVP are now instances!
 
Can we add the linkage from the DVP instance to the VehicleComponent? For example, vsso-signals:EngineCoolantTemp vsso-core:belongsToVehicleComponent vsso-signal:Engine)

danielwilms commented:
@felix-loesch, that's something we have to discuss later. It's possible only, if we instantiate the vehicle components as well...So I decided yesterday to discuss it first, before I implement it.

danielwilms commented:
Decision in meeting on 2022/05/16: creating the vehicle components as instances:
- possible to create subclass structure later on
- currently not sufficient information in VSS to have sub-classes in generation
- work well together with DVP as instances

danielwilms committed:
[VSSo] Vehicle components as instances

`Branch`es of VSS are now modeled as instances of `vsso-core:VehicleComponent`
and not as `rdfs:subClass` anymore.

Two challenges reside:
1. Shall there be an instance of `vsso:Vehicle` or punning of `vsso-core:Vehicle`?
   Challenge araises, when you use the `partOf` relationship.
2. Restrictions on Static Vehicle properties.

Signed-off-by: Daniel Wilms <Daniel.DW.Wilms@bmw.de>

danielwilms commented:
Meeting 2022/06/13:

Point from the PR:
> Shall there be an instance of `vsso:Vehicle` or punning of `vsso-core:Vehicle`?
   Challenge araises, when you use the `partOf` relationship.

Option 1: No root instance node of vehicle components
Option 2: Connecting to `vsso-core:Vehicle`, but then annotation property is necessary
Option 3: Having an instance vsso:Vehicle as über-VehicleComponent as root of the instances

we go for now with option 3 and discuss further.

danielwilms commented:
@danielwilms checks if option3 is implemented. If so, PR can be merged. Otherwise merged after implementation.

jdacoello commented:
@danielwilms to provide update on status of implementation of option 3.

danielwilms commented:
[Implementation done](https://github.com/COVESA/vss-tools/commit/c944887f613241f123c41b90f4596439ab6d93af). ttl file to be updated.

danielwilms committed:
Update vss ttl file

Updated ttl file, as implemented in vss-tools.
https://github.com/COVESA/vss-tools/commit/c944887f613241f123c41b90f4596439ab6d93af

Signed-off-by: Daniel Wilms <Daniel.DW.Wilms@bmw.de>

danielwilms commented:
ttl file added. Ready to merge.
